src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random;
+    private ColorSupplier colorSupplier;
+
+    public FigureSupplier() {
+        this.random = new Random();
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public Figure getRandomFigure() {
+        int generateRandomFigure = random.nextInt(5);
+        switch (generateRandomFigure) {
+            case 0:
+                return new Square(random.nextInt(100));
+            case 1:
+                return new Rectangle(random.nextInt(100),
+                        random.nextInt(100));
+            case 2:
+                return new RightTriangle(random.nextInt(100),
+                        random.nextInt(100));
+            case 3:
+                return new Circle(random.nextInt(100));
+            default:
+                return new IsoscelesTrapezoid(random.nextInt(100),
+                        random.nextInt(100), random.nextInt(50));
+        }
+    }
```

Comment:
you should create a separate method that will return the default figure with default parameteres
<img width="509" alt="Screenshot 2022-03-29 at 07 27 57" src="https://user-images.githubusercontent.com/60258373/160599027-e6fbcbc7-2c0f-4775-8549-c2c057e2de87.png">